templates/welcome.html

```diff
@@ -18,7 +18,7 @@
       
           <!-- Login Form -->
           <div style="margin:5%;">
-            Welcome : {{username}}
+            Welcome : {{username}}, You are successfully Registered
           </div>
           <div><a href="register">Register</a></div>
       
```

Comment:
Can we avoid the code duplication by using the templates better? For example, all the HTML and Head tags can be part of a generic template which can be imported?

Reply:
sure, will do that..

Reply:
done this one..